templates/LDA_Preprocessing.ipynb

```diff
@@ -1,15 +1,941 @@
 {
```

Comment:
<p>Same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Fair enough, although I don't think it's a bad idea to keep it in for now given that we ourselves had trouble finding a properly formatted example data source.  I imagine that the first few users will find it helpful to know which columns are required in their data, at least until the full working pipeline is established?</p>

Reply:
Sounds good, we can keep it until things are actually streamlined